_sass/_uswds-theme-custom-styles.scss

```diff
@@ -55,6 +55,11 @@ $logo-size-md: 2rem;
 }
 
 // Links
+p.usa-button a {
+  color: $color-white;
+  text-decoration: none;
+}
+
```

Comment:
I think we can take this out for now

Reply:
@igorkorenfeld thanks for catching that. PR updated with that button fix removed.